resources/mentors.py

```diff
@@ -319,14 +321,14 @@ def delete(self):
 
 
 class GetMultipleChoiceOptions(Resource):
-    # @jwt_required
-    def post(self):
```

Comment:
same here, these were split off and made posts for the reason stated above

Reply:
Got it